docker/entrypoint.sh

```diff
@@ -1,25 +1,34 @@
 #!/bin/sh
 if [ ! -e /etc/orchestrator-agent.conf.json ] ; then
+# @TODO think about full customization of orchestrator-agent.conf.json via ENV variables
+# @TODO think about mysql running not inside this docker
 cat <<EOF > /etc/orchestrator-agent.conf.json
 {
-    "SnapshotMountPoint": "/tmp",
-    "AgentsServer": "http://localhost",
-    "AgentsServerPort": ":3001",
-    "ContinuousPollSeconds" : 60,
-    "ResubmitAgentIntervalMinutes": 60,
-    "CreateSnapshotCommand": "echo 'no action'",
-    "AvailableLocalSnapshotHostsCommand": "echo 127.0.0.1",
-    "AvailableSnapshotHostsCommand": "echo localhost\n127.0.0.1",
-    "SnapshotVolumesFilter": "-my-snapshot-",
-    "MySQLDatadirCommand": "echo '~/tmp'",
+    "SnapshotMountPoint": "${SnapshotMountPoint:-/mysql-data}",
```

Comment:
I'd prefer using conventional `shell` naming, e.g. `${SNAPSHOT_MOUNT_POINT}`. This applies to all of the rest of the variables.